WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002AE7A2405A0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002AE7A240D10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002AE7A241040>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002AE7A241590>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002AE7A241AE0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002AE7A241F20>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session